class UserAlreadyExists(ValueError):
    pass


class RecordDoesNotExist(Exception):
    pass
